reframe/core/schedulers/slurm.py

```diff
@@ -233,7 +233,7 @@ def _get_reservation_nodes(self, reservation):
             reservation_nodes = node_match[1]
         else:
             raise JobError("could not extract the nodes names for "
```

Comment:
Can you change the message as well?

```suggestion
            raise JobError("could not extract the node names for "
```